core/gatsby-theme-docz/src/components/Playground/IframeWrapper.js

```diff
@@ -0,0 +1,23 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui'
+import Iframe from 'react-frame-component'
+
+import * as styles from './styles'
+
+const CODE_FONT = `<style type="text/css">@import url(https://fonts.googleapis.com/css?family=Inconsolata&display=swap);</style>`
+const INITIAL_IFRAME_CONTENT = `<!DOCTYPE html><html><head> ${CODE_FONT} <style> \n body { padding: 0; margin: 0; }  </style></head><body><div></div></body></html>`
```

Comment:
Yes, it would ! This felt a bit hacky, I'm not sure I like having the code_font styles and might remove them, because like you said it would go against what we're trying to do. 

Resetting the body margin and padding though shouldn't be too problematic I think. It's not necessary for sure but it felt ugly without it 😅 

I was trying to make the playground look exactly the same with or without the iframe but it's probably not a good idea.